presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -144,7 +144,7 @@ Array Functions
 
 .. function:: ngrams(array(T), n) -> array(array(T))
 
-    Returns ``n``-grams for the ``array``::
+    Returns ``n``-grams (sets of adjacent ``n`` elements) for the ``array``::
```

Comment:
"sets" -- for `([1, 2, 3, 4], 2)` input, ngrams will contain `[2, 3]` and not `[3, 2]`, so these are ordered
"sequences"? "sub-sequences"?

Also, might be worth saying something about the order of ngrams on the returned list. 
(whether it is left-to-right or unspecified)